tools/build_chain.sh

```diff
@@ -1611,12 +1615,17 @@ for line in ${ip_array[*]};do
         if [ -n "${ports_array[server_count]}" ];then
             read -r -a node_ports <<< "${ports_array[server_count]//,/ }"
             echo "node_ports ${node_ports[*]}" >>"${logfile}"
-            [ "${i}" == "0" ] && { set_value "${ip//./}_port_offset" 0; }
+            [ "${i}" == "0" ] && { set_value "${ip//[\.:]/_}_port_offset" 0; }
         fi
-        ip_list="${ip_list}node.${count}=${ip}:$(( $(get_value ${ip//./}_port_offset) + node_ports[0] ))
+        if [ -n "${use_ipv6}" ];then
+        ip_list="${ip_list}node.${count}=[${ip}]:$(( $(get_value ${ip//[\.:]/_}_port_offset) + node_ports[0] ))
     "
-        set_value ${ip//./}_count $(( $(get_value ${ip//./}_count) + 1 ))
-        set_value "${ip//./}_port_offset" $(( $(get_value "${ip//./}_port_offset") + 1 ))
+        else
+        ip_list="${ip_list}node.${count}=${ip}:$(( $(get_value ${ip//[\.:]/_}_port_offset) + node_ports[0] ))
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5837713)